cli/types/index.d.ts

```diff
@@ -62,6 +62,20 @@ declare namespace Cypress {
     isHeadless: boolean
   }
 
+  interface LocalStorage {
+    /**
+     * Called internally to clear `localStorage` in two situations.
+     *
+     * 1. Before every test, this is called with no argument to clear all keys.
+     * 2. On `cy.clearLocalStorage(keys)` this is called with `keys` as an argument.
+     *
+     * You should not call this method directly to clear `localStorage`; instead, use `cy.clearLocalStorage(key)`.
+     *
+     * @see https://on.cypress.io/clearlocalstorage
+     */
+    clear: (keys?: string[]) => void

```

Comment:
May be wrong here ...
But doesnt https://github.com/cypress-io/cypress/blob/master/packages/driver/src/cypress/local_storage.coffee#L11 indicate that there are 3 arguments to this function?

Reply:
Yeah, you can see I updated that code to only use 1 argument in this PR. ls and rs are always undefined.